.gitignore

```diff
@@ -7,3 +7,9 @@ tldextract_app/tldextract
 tldextract_app/web
 tldextract.egg-info
 .tox
+.idea
```

Comment:
Minor: can we keep this file focused on just what this project needs to ignore? Personal preferences e.g. editor or platform should be in a personal .gitignore.

Reply:
I didn't know I could do that - thanks!